docs/gke/configs/cluster-kubeflow.yaml

```diff
@@ -83,12 +83,20 @@ resources:
     #                 deployment in your project
     #          Project - This is the name of your GCP provided project.
     bootstrapperConfig: |
-      # Apps only apply if on GKE
+      registries:
```

Comment:
We don't need registries here once https://github.com/kubeflow/kubeflow/pull/951 merged.

Reply:
Ok. We can consider removing them once #951 is merged.